### ISSUE TITLE
Last Version

### ISSUE THREAD

**Issue by Riddler76** [lightphoenix2/LPX#24]
Hi,

Rocket have been updated.

LPX error

Failed to load LPX, unloading now... :System.TypeLoadException: Could not load type 'LIGHT.SQLPermission' from assembly 'LPX, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
  at Rocket.Core.Plugins.RocketPlugin.LoadPlugin () [0x00000] in <filename unknown>:0

And this spamming console

NullReferenceException: Object reference not set to an instance of an object - LIGHT.LIGHT.FixedUpdate ()

**Comment by weespin** [comment]
same.

**Comment by Riddler76** [comment]
http://puu.sh/p9CiX/9438b05310.dll

**Comment by Riddler76** [comment]
A friend have update the plugin ;)

**Comment by lammas112** [comment]
Doesnt work for me ...

**Comment by Riddler76** [comment]
Need to wait this friday have new update.

For moment the solution is use old rocket version : 917

**Comment by lammas112** [comment]
i just closed the server because some of stuff doesnt work like \* premissions ... so i will wait till everything is fixed...
 anyway thx

**Comment by weespin** [comment]
We're still waiting for your friend :yellow_heart:

**Comment by Riddler76** [comment]
Yeah we wait ^^ lightphoenix2 is at army for moment. IQBrod, come back tomorrow.

**Comment by weespin** [comment]
Yes, but phonenix just updated his own repo... :lollipop:

**Comment by Riddler76** [comment]
Epic update !
Auction cancel
Some fix <3
